phtc/templates/base.html

```diff
@@ -42,7 +41,13 @@
         <ul class="nav">
           <li>
             {% block brand %}
-                <a class="brand" href="/">{% if request.user.is_anonymous %}Home{% else %}My Dashboard{% endif %}</a>
+                <a class="brand"
```

Comment:
I'd approach this by replacing the anchor tags with a div. The style is defined generically as [.brand](https://github.com/ccnmtl/phtc/blob/master/media/css/main.css#L40), so there should be no side effects.